documentation/API_documentation/population-density-data-API-Readiness-Checklist.md

```diff
@@ -1,19 +1,19 @@
 # API Readiness Checklist
 
-Checklist for population-density-data 0.3.0-rc.1 in release r3.1
+Checklist for population-density-data 0.3.0 in release r3.2
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|------|
-|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/population-density-data.yaml) |
-|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   | [r3.2](https://github.com/camaraproject/Commonalities/releases/tag/r3.2) |
-|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   | [r3.2](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r3.2) |
-|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
-|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embed documentation into API spec - [link](/code/API_definitions/population-density-data.yaml) |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  N   |      |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](/code/Test_definitions/population-density-data.feature) |
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | [link](/code/API_definitions/population-density-data.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  | [r3.2](https://github.com/camaraproject/Commonalities/releases/tag/r3.2) |
```

Comment:
Commonalities release should be r3.3 - please confirm that is the case and if so update the release tag and link